docker/README_DEV.md

```diff
@@ -19,33 +19,80 @@ Once the image is built it needs to be tagged as `eclipse/rdf4j-workbench:${mave
 
 The simplest way to accomplish all of the above is to run the `build.sh` script.
```

Comment:
@jeenbroekstra might be better to have this sentence a bit higher up maybe.

Reply:
Good idea.

Reply:
Moved it to the start of the section